src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -510,6 +510,8 @@ export class RoslynLanguageServer {
             args.push('--extension', extensionPath);
         }
 
+        args.push('--devKitRazorOutputPath', 'C:/CodeHub/vscode-csharp/.razor');
```

Comment:
oops, I did not realize I am directly passing this.

I will fix this in a next commit